### PR TITLE
Multi-character language keys, fixes some language bugs i guess

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

title.

makes my life infinitely easier.

## Why It's Good For The Game

i hate not being able to do multikeyed languages.

## Changelog
:cl:
add: Multicharacter language keys.
tweak: Some language keys have been changed.
fix: Languages that suffered because of key conflicts no longer suffer the issue.
/:cl:

### CODE REVIEW

**Comment by Owai-Seek** [comment]
Does this resolve https://github.com/Skyrat-SS13/Skyrat13/issues/2452 ?

**Comment by BobJoga** [comment]
> Does this resolve #2452 ?

It does not.

**Comment by BobJoga** [comment]
j will fix and ezpand this pr tomorrow

**Comment by BobJoga** [comment]
> j will fix and ezpand this pr tomorrow

actually no, i didnt and i wont

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -347,7 +347,10 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 
 /mob/living/proc/get_message_language(message)
 	if(message[1] == ",")
-		var/key = message[1 + length(message[1])]
+		//skyrat edit - fucking hate this i want to use multichar keys
+		var/list/bruh = splittext(message, " ", 2)
```
Comment (Akrilla) [inline code/modules/mob/living/say.dm:351]: This doesn't feel too sane, could imagine this being null etc

Comment (BobJoga) [reply code/modules/mob/living/say.dm:351]: ok then what do i do

Comment (BobJoga) [reply code/modules/mob/living/say.dm:351]: ok wait hold on i'll just add a check for length

**Pull request merged**